### PR TITLE
Removed symfony BC layer for old symfony versions

### PR DESCRIPTION
**Pull request by core23**
I am targeting this branch, because this is BC.

## Changelog

```markdown
### Removed
 - Removed BC layer for old symfony versions
```

### CODE REVIEW

File: src/Block/Social/PinterestPinButtonBlockService.php [hunk 1]
```diff
@@ -48,25 +49,25 @@ public function configureSettings(OptionsResolver $resolver)
...
                     'required' => false,
                     'label' => 'form.label_description',
                 ]],
-                ['size', 'integer', [
+                ['size', IntegerType::class, [
                     'required' => false,
                     'label' => 'form.label_size',
                 ]],
-                ['shape', 'choice', [
+                ['shape', ChoiceType::class, [
```
Comment (jordisala1991) [inline src/Block/Social/PinterestPinButtonBlockService.php]: This file is missing some uses to load all that classes

Comment (core23) [reply src/Block/Social/PinterestPinButtonBlockService.php]: Fixed

**Comment by jordisala1991** [comment]
Build fixed in: https://github.com/sonata-project/SonataSeoBundle/pull/228 (let's hope)

**Comment by core23** [comment]
Please review again @jordisala1991

**Comment by jordisala1991** [comment]
Thank you @core23

**Pull request merged**